lib/octokit/error.rb

```diff
@@ -34,9 +34,17 @@ def self.from_response(response)
       end
     end
 
+     def build_error_context
+       rate_limited_errors = [Octokit::TooManyRequests, Octokit::AbuseDetected]
```

Comment:
Can we make this into a constant?

Reply:
Sure, moved the constants to the bottom due to undefined issues.